src/api/transactions.js

```diff
@@ -1,11 +1,16 @@
 import transactions from './data.json';
-
 const TIMEOUT = 250;
 
 export default {
-	getTransactions(cb) {
-		setTimeout(() => {
-			cb(transactions);
-		}, TIMEOUT);
+	getTransactions() {
+		return new Promise((resolve, reject) => {
+			setTimeout(() => {
+				if (transactions.message === 'OK'){
+					resolve(transactions.result);
+				}else{
+					reject(transactions);
+				}
+			}, TIMEOUT);
```

Comment:
Why are we using a timeout?

Reply:
Because the blockchain functionality is not deployed, we're using it to emulate a backend request time